plugin_tooling/src-tests/com/github/rustdt/tooling/RustBuildOutputParserClassicTest.java

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+ * Copyright (c) 2014 Bruno Medeiros and other Contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Bruno Medeiros - initial API and implementation
+ *******************************************************************************/
+package com.github.rustdt.tooling;
+
+
+import static melnorme.utilbox.core.Assert.AssertNamespace.assertFail;
+import static melnorme.utilbox.core.CoreUtil.listFrom;
+import static melnorme.utilbox.status.Severity.ERROR;
+import static melnorme.utilbox.status.Severity.WARNING;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import melnorme.lang.tooling.common.ToolSourceMessage;
+import melnorme.utilbox.core.CommonException;
+
+
+public class RustBuildOutputParserClassicTest extends RustBuildOutputParserTest {
```

Comment:
RustBuildOutputParserTest.java contained all the tests for the output parser. I moved the code specific to the 'classical' format (rustc 1.11 or below) to this file and kept the general code in RustBuildOutputParserTest.java.